src/views/DatasetManager/components/CreateDatasetButton/components/DatasetCreationParameters/DatasetCreationParameters.js

```diff
@@ -0,0 +1,147 @@
+      const parameterId = parameter.id;
+      const inputType = parameter.varType;
+      const parameterKey = isDataSourceTypeRunner(dataSourceType) ? `${dataSourceType}.${parameterId}` : parameterId;
+
+      return (
+        <Controller
+          key={parameterKey}
+          name={parameterKey}
+          rules={{ required: true }}
+          render={({ field }) => {
+            const { value, onChange } = field;
+            if (inputType === 'string') {
+              const input = (
+                <Grid item xs={12} sx={{ pt: 1 }}>
+                  <BasicTextInput
+                    id={parameterId}
+                    key={parameterId}
+                    label={t(TranslationUtils.getParameterTranslationKey(parameterId), parameterId)}
+                    tooltipText={t(TranslationUtils.getParameterTooltipTranslationKey(parameterId), '')}
+                    size="medium"
+                    value={value ?? ''}
+                    changeTextField={(newValue) => onChange(newValue)}
+                  />
+                </Grid>
+              );
+              return input;
+            } else if (inputType === 'enum') {
+              const textFieldProps = {
+                disabled: false,
+                id: `enum-input-${parameterId}`,
+              };
+              const enumValues = getParameterEnumValues(parameterId);
+              return (
+                <Grid item xs={6} sx={{ pt: 2 }}>
+                  <BasicEnumInput
+                    id={parameterId}
+                    key={parameterId}
+                    label={t(TranslationUtils.getParameterTranslationKey(parameterId), parameterId)}
+                    tooltipText={t(TranslationUtils.getParameterTooltipTranslationKey(parameterId), '')}
+                    value={value ?? enumValues?.[0]?.key ?? ''}
+                    changeEnumField={onChange}
+                    textFieldProps={textFieldProps}
+                    enumValues={enumValues}
+                  />
+                </Grid>
+              );
+            } else if (inputType === '%DATASETID%') {
+              const { value, onChange } = field;
+              return (
+                <Grid item xs={12} sx={{ pt: 1 }}>
+                  <UploadFile
+                    id={parameterId}
+                    key={parameterId}
+                    labels={getUploadFileLabels(parameterId)}
+                    tooltipText={t(TranslationUtils.getParameterTooltipTranslationKey(parameterId), '')}
+                    handleUploadFile={(event) => FileManagementUtils.prepareToUpload(event, onChange)}
+                    editMode={true}
+                    handleDeleteFile={() => onChange({})}
+                    file={value ?? {}}
+                    acceptedFileTypes={getDefaultFileTypeFilter(parameterId)}
+                  />
+                </Grid>
+              );
+            } else {
+              console.error('DataSource parameter vartype unknown');
```

Comment:
Can you add the value of the unknown vartype in the console message please? Also, the parameter id would help :slightly_smiling_face: